cmd/ignition2rpm/README.md

```diff
@@ -0,0 +1,24 @@
+# ignition2rpm
+
+# ! Do not depend on this or use this for anything, it is a science experiment !
+
+Tooling to convert ignition to RPM files for use in rpm-ostree images
+
+This really just uses rpmpack ( github.com/google/rpmpack ) to package CoreOS's ignition format into an RPM file
+
+
+Process a local ignition into an rpm: 
+```
+ignition2rpm --config machine_config.ign --output ./machine_config.rpm 
+```
+
+Process a remote ignition into an rpm:
+```
+ignition2rpm --config http://host.example.com/machine_config.ign --output ./machine_config.rpm 
+```
+
+Also works with MachineConfig ( or at least it seems to, but I haven't tested this exhaustively ). 
+
+Things to be aware of: 
+	- It relocates some files from /usr/local to /var/usrlocal to make rpm-ostree happy 
```

Comment:
@jkyros Did we ever find a canonical source of these? It would probably be worth it to include a reference to it even as a comment.

Reply:
Yes-ish? I found them in the rpm-ostree source: 

- The root ones: 
https://github.com/coreos/rpm-ostree/blob/d96748c353b3837124add5e6ebad083966533e04/rust/src/composepost.rs#L67
- Some other ones: 
https://github.com/coreos/rpm-ostree/blob/d96748c353b3837124add5e6ebad083966533e04/rust/src/composepost.rs#L749

Reply:
Cool! It's interesting that they have a special case for Vagrant: https://github.com/coreos/rpm-ostree/blob/d96748c353b3837124add5e6ebad083966533e04/rust/src/composepost.rs#L752.